CRM/Civiquickbooks/Form/Settings.php

```diff
@@ -139,11 +139,25 @@ public function getFormSettings() {
   public function saveSettings() {
     $settings = $this->getFormSettings();
     $values = array_intersect_key($this->_submittedValues, $settings);
-    $previousValues = CRM_Quickbooks_APIHelper::getQuickBooksCredentials();
+
+    // Fix for unsetting a checkbox. When setting a checkbox,
```

Comment:
This section that adds the boolean settings fix looks good, although it looks as though it actually belongs to a different PR